docs/source/guide/webhook_reference.md

```diff
@@ -15,6 +15,14 @@ Webhooks sent from Label Studio include the following:
 | --- | --- |
 | action | Details the action that the event represents. |
 | project | Included only for task and annotation events. Details about the project. |
+| task_number | Total number of tasks in the project. |
+| finished_task_number | Total number of annotated or skipped tasks in the project. |
+| total_predictions_number | Total number of predictions for the project. |
+| total_annotations_number | Total number of annotations, skipped tasks, and ground truth annotations in the project. Can be different from the total number of annotated tasks. |
```

Comment:
and here too

Reply:
total_annotations_number = serializers.IntegerField(default=None, read_only=True,
                                                    help_text='Total annotations number in project including '
                                                              'skipped_annotations_number and ground_truth_number.')
in projects/serializers.py

this is why I listed skipped annotations here